packages/management/src/app/_layout-client.tsx

```
@@ -5,6 +5,8 @@ import '@lib/style/main.css';
 import { ColorSchemeScript, MantineProvider, createTheme } from '@mantine/core';
 import { Notifications } from '@mantine/notifications';
 import { useEffect, useState } from 'react';
+import { usePathname, useRouter } from 'next/navigation';
```

Biome (1.9.4)

[error] The current dependency isn't specified in your package.json.

This could lead to errors.
Add the dependency in your manifest.

(lint/correctness/noUndeclaredDependencies)